scripts/wildfly/install-wildfly.sh

```diff
@@ -12,7 +12,8 @@
 cd "$(dirname "$0")" || exit
 
 # WildFly version should match the version used in pom.xml
-export WILDFLY_VERSION=30.0.1.Final
+# Please follow the instructions in 'updatingDependencies.md' when upgrading WildFly.
+export WILDFLY_VERSION=31.0.0.Final
```

Comment:
No. I really don't see how. Also, there are other manual steps you need to take anyhow. It would take a lot of work to try to automate this, if possible at all.